src/phases/execution/ClusteredCollection.yml

```diff
@@ -12,8 +12,21 @@ GlobalDefaults:
   Collection: &Collection Collection0
   Namespace: &Namespace test.Collection0
   DocumentCount: &docCount 1000000
-  MaxPhases: &MaxPhases 7
-
+  MaxPhases: &MaxPhases 9
+  SmallId: &SmallId {^Join: {array: [
+    { ^FormatString: {"format": "%07.0f", "withArgs": [
+      {^RandomDouble: {
+        min: 1,
+        max: {^NumExpr: {withExpression: "(2 * docCount) / 10", andValues: {docCount: *docCount}}}}}]}},
+    *PaddingGenerator]}}
+  BigId: &BigId {^Join: {array: [
+    { ^FormatString: {"format": "%07.0f", "withArgs": [
+      {^RandomDouble: {
+        min: {^NumExpr: {withExpression: "(8 * docCount) / 10", andValues: {docCount: *docCount}}},
+        max: *docCount}}]}},
+    *PaddingGenerator]}}
+  SmallSecondaryKey: &SmallSecondaryKey {^Join: {array: ["1", {^FastRandomString: {length: 5, alphabet: "0123456789"}}]}}
```

Comment:
What's the benefit of having a random boundary each time the perf test is run?

Reply:
That is compared with the fixed boundaries for the _id field.

Reply:
It is not fixed for the whole test, it is different boundary for each query.

I just used this section to define this large expressions to make phase definitions more readable.

Reply:
Moved them from GlobalDefaults to ValueGenerators field for clarity

Reply:
It shouldn't be, because I think there are enough queries made for statistics to stabilize
If we see this tests to be unstable, we can narrow the random ranges.

There still has to be some randomness, so we won't cache exact queries.